### PR TITLE
Input

### PR DESCRIPTION
**Pull request by raieegulhane**
image
image

### CODE REVIEW

File: components/input/input.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    <link rel="stylesheet" href="/src/style.css">
+    <link rel="stylesheet" href="/assets/navbar.css">
+    <link rel="stylesheet" href="/assets/general.css">
+    <link rel="stylesheet" href="/assets/sidebar.css">
+    <link rel="stylesheet" href="/assets/component-showcase.css">
+    <link rel="stylesheet" href="/components/flex/flex.css">
+    <link rel="stylesheet" href="/components/grids/grids.css">
+    <link rel="stylesheet" href="/components/button/button.css">
+    <link rel="stylesheet" href="/components/lists/lists.css">
+    <link rel="stylesheet" href="/components/input/input.css">
```
Comment (anurag-kkumarr) [inline components/input/input.html:16]: Import all these CSS in a single file and Import that css file.

File: components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    <script src="https://kit.fontawesome.com/10a3e21499.js" crossorigin="anonymous"></script>
+
+    <title>MatrixUI | Input</title>
+</head>
+<body>
+
+    <!------------ navbar ------------>
+    <header class="navbar_container">
+        <nav class="navbar flex flex-row flex-centered">
+            <a class="link-noDecoration" href="/index.html">
```
Comment (anurag-kkumarr) [inline components/input/input.html:27]: Try to use one specific naming convention. Either use 'dash' or 'underscore'. Don't mix both

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]

**Review by anurag-kkumarr — COMMENTED** [review]
Else everything looks good

**Review by Pratham82 — COMMENTED** [review]
Good to go

**Pull request merged**